src/linuxdeploy.cpp

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
+#include <boost/filesystem/path.hpp>
+
+#include <linuxdeploy/core/appdir.h>
+#include <linuxdeploy/core/log.h>
+#include <linuxdeploy/util/util.h>
+
+#include "linuxdeploy.h"
+
+using namespace linuxdeploy::core;
+
+using namespace linuxdeploy::core::log;
+using namespace linuxdeploy::util;
+namespace bf = boost::filesystem;
+
+namespace linuxdeploy {
```

Comment:
No need for a namespace here, is it? These functions don't belong to the library, but only the "frontend" that is `main.cpp`.

Reply:
are namespaces only for libraries? I guess that their purpose is not only to avoid name classes but also to partition code. That's why used a namespace here to indicate that this is part of the linuxdeploy logic

Reply:
Call it "personal preference" then. I always thought "put library stuff in namespaces, and non-library, frontend code into root namespace". But I don't insist on that.

Reply:
Right now it will not make any harm. Let's save this for a future review.